redwood-protocol-compose/src/commonMain/kotlin/app/cash/redwood/protocol/compose/ProtocolMismatchHandler.kt

```diff
@@ -48,5 +48,23 @@ public interface ProtocolMismatchHandler {
         )
       }
     }
+
+    /** A [ProtocolMismatchHandler] which prints out info for all callbacks. */
+    @JvmField
+    public val Logging: ProtocolMismatchHandler = object : ProtocolMismatchHandler {
```

Comment:
If we're going to add something to the Compose `ProtocolMismatchHandler` we should probably add it to the Widget `ProtocolMismatchHandler` to keep API symmetry.